corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -466,11 +466,23 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
             </div>
           </div>
           {% endif %}
+          {% if request|toggle_enabled:'SPLIT_SCREEN_CASE_SEARCH' and not app.split_screen_dynamic_search%}
```

Comment:
why would this not be configurable with dynamic search?

Reply:
Dynamic search already includes "search on clear" behavior. So if dynamic search is enabled, there's no need to also specifically enabled "search on clear" so the UI toggle is hidden.

Reply:
cool I missed that detail sounds good